python/lsst/ts/standardscripts/maintel/stop_rotator.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["StopRotator"]
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+
+class StopRotator(salobj.BaseScript):
+    """A script that executes stop_rotator method for `MTCS`
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    Stopping rotator: Before issuing stop command.
+    Done: After issuing stop command.
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(index=index, descr="MTCS stop rotator.")
+
+        mtcs_usage = None if add_remotes else MTCSUsages.DryTest
+
+        self.mtcs = MTCS(domain=self.domain, intended_usage=mtcs_usage, log=self.log)
```

Comment:
I would suggest moving the creating of the MTCS object into the `configure` method. So, here you would do:

```
self.mtcs = None
```

Then, in the `configure` method you would do:

```
if self.mtcs is None:
    self.mtcs = MTCS(domain=self.domain, log=self.log)
```

Then remove the `add_remotes` from the class initialization and the `mtcs_usage` above. 


In the unit test you can do something like this:

```
    async def basic_make_script(self, index):
        self.script = StopRotator(index=index)

        self.script.mtcs = unittest.mock.AsyncMock()
        self.script.mtcs.configure_mock(tel_settle_time=3.0)  # need this for the set_metadata method.
        return (self.script,)
```

Then, you make a unit test like this:

```
    async def test_run(self):
        async with self.make_script():
            await self.configure_script()

            await self.run_script()

            self.script.mtcs.stop_rotator.assert_awaited_once()
```